packages/appkit-utils/src/ethers/InjectedProvider.ts

```diff
@@ -0,0 +1,19 @@
+import type { Provider } from '@reown/appkit-controllers'
+
+import { EthersProvider } from './EthersProvider.js'
+
+export class InjectedProvider extends EthersProvider<Provider> {
```

Comment:
What does this provider do exactly? In my Headless development I've realized we still have "Browser wallet" thing that we just filtering it out. I guess this causes that? Do we really need this injected provider?